eslint.config.js

```diff
@@ -47,6 +49,13 @@ export default [
         'error',
         { type: 'element', prefix: 'app', style: 'kebab-case' },
       ],
+      'header/header': [
+        2,
+        'block',
+        ['*', ' * @license Apache-2.0', ' * @copyright The GHGA Authors', ' '],
```

Comment:
Is there a reason why you put the license first? Usually the copyright comes first, then the license.

Reply:
Also, shouldn't we use `@license SPDX-License-Identifier: Apache-2.0` for better machine readability?

Reply:
For licenses I think that's not recommended because @license already specifies that matter. If I add it, JSDoc throws this error: Invalid JSDoc @license: "SPDX-License-Identifier: Apache-2.0"; expected SPDX expression: https://spdx.org/licenses/
But I will swap the order

Reply:
The SPDX tag should probably also not go into a block comment, but the line must start with `/*` or `//`

https://spdx.dev/learn/handling-license-info/#how

Reply:
https://jsdoc.app/tags-license hier verwenden sie genau das Beispiel auf der JSDoc Seite als Block Kommentar, daran hatte ich mich orientiert. Aber wir können das machen wie wir wollen denke ich, weil ich vermute, dass die ganzen Parser wahrscheinlich sowieso nach den einzelnen Optionen in der SPDX List suchen.

Reply:
Just for the records: [JSDoc](https://jsdoc.app/tags-license) uses block comments, [SPDX](https://spdx.dev/learn/handling-license-info/#how) tags use line comments, and they use different tags (`@license` vs `SPDX-License-Identifier:`).

This makes it impossible to combine them in one line.

We could add both types of license tags in separate lines, but this would be ugly and redundant. So we decided to include the JSDoc style license tag only.